docs/YIP/YIP-0003-pretrade-risk-checks.md

```diff
@@ -9,10 +9,29 @@ In Work
 We have to develop the mechanism that will limit users possibility to create orders that will put parties at risk of not been able to cover the net exposure by the colleteral. Exposing the parameters by API so it will be visible and understood by the user of the system. Like a Buying power = colleteral * levarage - open order amount, and Equity = colleteral + net exposure.
 https://www.b2bits.com/trading_solutions/pre-trade-risk-check-solutions
 
+
```

Comment:
let's name this YIP as `Pre-Trade risk checks vol.1` because we should have more risk checks with net exposure